src/System.Net.Sockets.Legacy/tests/FunctionalTests/DisconnectAsync.cs

```diff
@@ -15,6 +15,7 @@ public void OnCompleted(object sender, SocketAsyncEventArgs args)
         }
 
         [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
```

Comment:
Both good suggestions.

@stephentoub these tests are Windows-specific b/c `Disconnect` cannot be supported on other platforms.

@CIPop I'll add the suggested tests.